native/src/components/ExportEventButton.tsx

```diff
@@ -0,0 +1,117 @@
+import moment from 'moment'
+import React, { ReactElement, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { Linking, Platform } from 'react-native'
+import RNCalendarEvents, { Calendar } from 'react-native-calendar-events'
+import { Button } from 'react-native-elements'
+import { useTheme } from 'styled-components'
+
+import { EventModel } from 'api-client'
+
+import useSnackbar from '../hooks/useSnackbar'
+import CalendarChoice from './CalendarChoice'
+
+type ExportEventButtonType = {
+  event: EventModel
+}
+
+const ExportEventButton = ({ event }: ExportEventButtonType): ReactElement => {
+  const { t } = useTranslation('events')
+  const theme = useTheme()
+  const showSnackbar = useSnackbar()
+
+  const [eventExported, setEventExported] = useState<boolean>(false)
+  const [showCalendarChoiceOverlay, setShowCalendarChoiceOverlay] = useState<boolean>(false)
+  const [calendars, setCalendars] = useState<Calendar[]>()
+
+  const saveEventToCalendar = async (calendarId: string | undefined): Promise<void> => {
```

Comment:
We should stick to either save or export I think:
```suggestion
  const exportEvent = async (calendarId: string | undefined): Promise<void> => {
```